libs/fssr/octree.cc

```diff
@@ -363,10 +364,16 @@ Octree::influence_query (math::Vec3d const& pos, double factor,
      * - Note: Nodes can contain samples with scale values much smaller than
      *   node_size. This is because the octree depth is limited.
      */
-
-    math::Vec3d node_center;
-    double node_size;
-    this->node_center_and_size(iter, &node_center, &node_size);
```

Comment:
Yes, it's used in [`Octree::find_node_descend()`](https://github.com/andre-schulz/mve/blob/1fb6227eb81ede049d96f24e71c37cf5df5a2489/libs/fssr/octree.cc#L246).